projects/opentrain/gtfs_pipeline/heatmap/HeatMap.java

```diff
@@ -0,0 +1,325 @@
+package projects.opentrain.gtfs_pipeline.heatmap;
+
+import com.projects.opentrain.gtfs_pipeline.Protos.Stop;
+import com.projects.opentrain.gtfs_pipeline.Protos.StopTime;
+import com.projects.opentrain.gtfs_pipeline.Protos.Trip;
+
+import javax.imageio.ImageIO;
+import java.awt.*;

```

Comment:
We don't use `*` imports, so please replace that with the individual class imports.